WordPress/Classes/ViewRelated/Pages/PageMenuViewModelTests.swift

```diff
@@ -1,205 +0,0 @@
-import Nimble
-import XCTest
-@testable import WordPress
-
-final class PageMenuViewModelTests: CoreDataTestCase {
-
-    func testPublishedPageButtonsWithBlazeEnabled() {
-        // Given
-        let page = PageBuilder(mainContext, canBlaze: true)
-            .withRemote()
-            .with(status: .publish)
-            .build()
-        let viewModel = PageMenuViewModel(
-            page: page,
-            isSiteHomepage: false,
-            isSitePostsPage: false,
-            isJetpackFeaturesEnabled: true,
-            isBlazeFlagEnabled: true
-        )
-
-        // When & Then
-        let buttons = viewModel.buttonSections
-            .filter { !$0.buttons.isEmpty }
-            .map { $0.buttons }
-        let expectedButtons: [[AbstractPostButton]] = [
```

Comment:
Removing the tests that were covering "configuration". If the complexity of the code under tests and the tests themselves is equal, it's a waste to maintain them.